### DISCUSSION TITLE
How to delete one occurence

### DISCUSSION THREAD

**Discussion by Cer41Tom** [Etar-Group/Etar-Calendar#1527] [General]
When I create & save a recurring event, it sometimes happens that I need to remove one particular occurence.
However, this is not possible - I am offered to choose between deleting "This and future events", or "All events" (neither of which I want).
Is this a feature? How can I delete one specific occurence then?
Thanks a lot, Tom

**Comment by Stephen443** [comment]
I miss this feature too.
Is there any hope that it might get implemented?
